docs/data/joy/getting-started/tutorial/tutorial.md

```diff
@@ -0,0 +1,233 @@
+# Tutorial
+
+<p class="description">We'll learn how to create a login page using Joy components.</p>
+
+## Topics covered
+
+1. Importing components.
+2. Building a log in page using Joy components.
+3. What global variants look like.
+4. Toggling dark mode
+
+## What's not covered?
+
+- Theming and component customization.
+- Features explanation.
+
```

Comment:
Thinking again, I think we could remove all of this. As the page is small, is relatively quick to skim through the content and see what it covers—the table of contents helps with that too. As far as what is not approached, we could use the "Where to go from here" to broaden up to these topics.

Reply:
My purpose is to set expectations before going through the tutorial because I can imagine that they will have questions along the way which we don't plan to explain the details in this tutorial.

I'd love to hear @samuelsycamore feedback on this and goes with his opinion.